### PR TITLE
Add pip invocations

### PR DESCRIPTION
**Pull request by OttoAndrey**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ We follow [Semantic Versions](https://semver.org/).
 - Improve `django.createsuperuser` command with verbose name settings
 - Fix k8s.set-context when user is not logged in for first time
 
+## 0.8.0
+
+- Add pip invocations
```
Comment (TheSuperiorStanislav) [inline CHANGELOG.md:12]: Please move it under #unreleased

File: README.md [hunk 2]
```diff
@@ -692,3 +695,14 @@ using [poetry up plugin](https://github.com/MousaZeidBaker/poetry-plugin-up).
...
+
+### pip
+
+#### pip.install-dependencies
+
+Install dependencies via pip.
+
+#### pip.compile-dependencies
+
+Compile dependencies via
```
Comment (TheSuperiorStanislav) [inline README.md:707]: Please describe available settings

File: README.md [hunk 3]
```diff
@@ -692,3 +695,23 @@ using [poetry up plugin](https://github.com/MousaZeidBaker/poetry-plugin-up).
...
+
+### pip
+
+#### pip.install-dependencies
+
+Install dependencies via pip.
+
+Settings:
+
+* `dependencies_folder` path to folder with dependencies files (Default: `requirements`)
```
Comment (TheSuperiorStanislav) [inline README.md:707]: You sure?

File: saritasa_invocations/_config.py [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import collections.abc
 import dataclasses
+from pathlib import Path
```
Comment (TheSuperiorStanislav) [inline saritasa_invocations/_config.py:3]:
```suggestion
import pathlib
```

Comment (TheSuperiorStanislav) [reply saritasa_invocations/_config.py:3]: Why not just use `str`?

**Review by TheSuperiorStanislav — APPROVED** [review]

**Review by MrShirokow — APPROVED** [review]

**Pull request merged**